third_party/terraform/tests/resource_compute_instance_template_test.go

```diff
@@ -1868,3 +1888,29 @@ resource "google_compute_instance_template" "foobar" {
 	}
 }`, acctest.RandString(10), enableSecureBoot, enableVtpm, enableIntegrityMonitoring)
 }
+
+func testAccComputeInstanceTemplate_enableDisplay() string {
+	return fmt.Sprintf(`
+data "google_compute_image" "my_image" {
+	family  = "centos-7"
+	project = "gce-uefi-images"
+}
+
+resource "google_compute_instance_template" "foobar" {
+	name = "instancet-test-%s"
+	machine_type = "n1-standard-1"
+	can_ip_forward = false
+
+	disk {
+		source_image = "${data.google_compute_image.my_image.self_link}"
+		auto_delete = true
+		boot = true
+	}
+
+	network_interface {
+		network = "default"
+	}
+
+	enable_display = "true"
```

Comment:
You don't need the double quotes around this as Terraform syntax supports booleans.